src/views/Viewer.vue

```diff
@@ -1276,7 +1276,8 @@ export default {
 	}
 
 	&.theme--dark:deep(.button-vue--vue-tertiary) {
-		&:hover {
+		&:hover,
+		&.action-item__menutoggle {
 			background-color: rgba(255, 255, 255, .08) !important;
```

Comment:
🤔 base state should be `background-color: transparent` like the other buttons with this background-color kept for hover/focus
![image](https://github.com/nextcloud/viewer/assets/24800714/c997fcdf-8782-413f-88e5-d1f09433f1ca)